cves/CVE-2017-16840.yml

```diff
@@ -357,11 +390,13 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 120
+CWE_note: |
+  Since there was not proper padding, it could lead to a buffer overflow as 
+  the Haar wavelets need to check outside of the frame.
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Frame Slice
```

Comment:
This nickname is too generic. Frame slicing is a standard operation in codecs. Please think of a more specific or original name.